package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "planet-client",
-  "version": "0.14.0",
+  "version": "0.15.1",
```

Comment:
Please don't update the published version in a pull request.  This lets us merge multiple branches per release.